src/server_manager/web_app/main.ts

```diff
@@ -103,6 +114,10 @@ document.addEventListener('WebComponentsReady', () => {
   // NOTE: this cast is safe and allows us to leverage Polymer typings since we haven't migrated to
   // Polymer 3, which adds typescript support.
   const appRoot = document.getElementById('appRoot') as unknown as AppRoot;
+
+  const filteredLanguageDefs = Object.values(SUPPORTED_LANGUAGES)
+      .filter((languageDef) => languageDef.id !== 'zh');
```

Comment:
Let's remove zh from the list. And maybe remove the message file too.
@alalamav could you weigh in?

Reply:
Keeping the more specific variant (zh-CN) sounds good to me. We should also update the translation configuration to prevent additional translations (happy to do it).

Reply:
Sounds good. I removed the 'zh' language code and associated messages file.

@alalamav I'll sync with you on the translation configuration so that I can learn how that works.